php/login.php

```diff
@@ -165,29 +107,114 @@ function auth($email, $password) {
 }
 
 /**
- * Helper function for parsing Drupal headers for login.
+ * Parse form build ID
```

Comment:
Can you double check,  but I think all of these helper methods can be removed. I think they were from a time when we had to screenscrape the login form... But now we use an API command.